metadata_manager/core.py

```diff
@@ -207,6 +207,7 @@ def __init__(self, remotes_json_path: str,
         )
         self.__task__runner = TaskRunner(tasks=tasks)
         self.repo = ap_repo
+        self.reload_remotes_json()
```

Comment:
Are you *sure* you want to do this in an init function?

It can seriously stuff with the order in which you want to do things with objects and can make re-use problematic.

Reply:
Ummm... Okay. Your concern seems to be valid. I will keep it in app.py for now.